idealingua-v1/idealingua-v1-runtime-rpc-http4s/src/main/scala/izumi/idealingua/runtime/rpc/http4s/HttpRequestContext.scala

```diff
@@ -1,6 +1,6 @@
 package izumi.idealingua.runtime.rpc.http4s
 
-import org.http4s.AuthedRequest
+import org.http4s.Request
 
 // we can't make it a case class, see https://github.com/scala/bug/issues/11239
-class HttpRequestContext[F[_], Ctx](val request: AuthedRequest[F, Ctx], val context: Ctx)
+class HttpRequestContext[F[_]](val request: Request[F], val context: Any)
```

Comment:
@Caparow Might as well delete it, it's not used anywhere now.

Reply:
Yeah, I missed it
Thank you